File: Leetcode-Easy/485-MaxConsecutiveOnes/MaxConsecutiveOnes.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = 0;
        int countOfOne = 0;
        for (int num:nums)
        {
            if (num == 0)
            {
                if (max<countOfOne)
                {
                    max=countOfOne;
                }
                countOfOne=0;
            }
            else
            {
                countOfOne++;
            }
        }
        return max<countOfOne?countOfOne:max;
    }
}

File: Leetcode-Easy/485-MaxConsecutiveOnes/MaxConsecutiveOnes.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int count =0;
        int res = 0;
        for(int i=0;i<nums.size();i++){
            if(nums[i]==0){
                count=0;
            }
            else{
                count ++;
                res=max(count,res);
            }
        }
        
        return res;
    }
};